python/katana/local/convert_graph.py

```diff
@@ -0,0 +1,114 @@
+from typing import Collection, Dict, Optional, Union
+
+import numba
+import numpy as np
+
+import katana.local
+from katana.native_interfacing.buffer_access import to_numpy
+
+
+def from_adjacency_matrix(adjacency: np.ndarray, property_name: str = "weight") -> katana.local.Graph:
+    """
+    Convert an adjacency matrix with shape (n_edges, n_edges) into a :py:class:`~katana.local.Graph` with the
```

Comment:
(m_nodes, n_edges) ?

Reply:
why is the second part not edges? i understand it's an array of nodes but it's basically the number of edges for a node?

Reply:
Because we are expecting an adjacency matrix. So it's square with side length of the number of nodes. https://en.wikipedia.org/wiki/Adjacency_matrix